From 122c018d085e4f773cd81f226623202a3059e370 Mon Sep 17 00:00:00 2001
From: Arnd Bergmann <arnd@arndb.de>
Date: Tue, 13 Feb 2024 10:53:18 +0100
Subject: [PATCH 03/18] NOTCBL: FROMLIST: ALSA: arm: remove an unused variable

The only user of a variable got removed a while ago, but I only now
ran into this on randconfig builds with clang:

sound/arm/aaci.c:742:15: error: unused variable 'aaci' [-Werror,-Wunused-variable]
  742 |         struct aaci *aaci = card->private_data;

Remove the unnecessary variable.

Fixes: 793e0fca25fa ("ALSA: arm: Remove superfluous snd_pcm_suspend*() calls")
Signed-off-by: Arnd Bergmann <arnd@arndb.de>
Link: https://lore.kernel.org/r/20240213095336.453562-1-arnd@kernel.org
---
 sound/arm/aaci.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sound/arm/aaci.c b/sound/arm/aaci.c
index f64896564728..c3340b8ff3da 100644
--- a/sound/arm/aaci.c
+++ b/sound/arm/aaci.c
@@ -739,7 +739,6 @@ static const struct snd_pcm_ops aaci_capture_ops = {
  */
 static int aaci_do_suspend(struct snd_card *card)
 {
-	struct aaci *aaci = card->private_data;
 	snd_power_change_state(card, SNDRV_CTL_POWER_D3cold);
 	return 0;
 }
-- 
2.43.2

